<?xml version="1.0"?>
<launch>
  <!--arg name="frame_id" default="/scan"/-->
  <arg name="frame_id" default="/laser_cloud"/>
  <arg name="scanner_ip" default="192.168.1.20"/>
  <arg name="scan_frequency" default="20"/>
  <arg name="samples_per_scan" default="2400"/>

  <!-- R2000 Driver -->
  <node pkg="pepperl_fuchs_r2000" type="r2000_node" name="r2000_node" respawn="true" output="screen">
    <param name="frame_id" value="$(arg frame_id)"/>
    <param name="scanner_ip" value="$(arg scanner_ip)"/>
    <param name="scan_frequency" value="$(arg scan_frequency)"/>
    <param name="samples_per_scan" value="$(arg samples_per_scan)"/>
  </node>
  <!--node pkg="tf" type="static_transform_publisher" name="scan_frame" args="0.0 0.0 0.0 0.0 0.0 0.0 map scan 100" />-->
  <node pkg="tf" type="static_transform_publisher" name="cloud_frame" args="0.0 0.0 0.0 0.0 0.0 0.0 map laser_cloud 100" />
  <!-- RVIZ -->
  <!--<node pkg="rviz" type="rviz" name="$(anon rviz)" respawn="false" output="screen" args="-d $(find pepperl_fuchs_r2000)/rviz/rviz.rviz"/>-->
  <!--node pkg="rviz" type="rviz" name="$(anon rviz)" respawn="false" output="screen" args="-d $(find pepperl_fuchs_r2000)/rviz/pointcloud.rviz"/-->
</launch>
